cmake/one_core_compliance.cmake

```diff
@@ -0,0 +1,36 @@
+#has to be set before project
+if(MSVC_ONE_CORE_COMPLIANCE)
+    message([STATUS] "Building OneCore Compliant Binaries")
+    if(DEFINED ENV{varBuildWdkVer})
+        # For QB or manual setting - QB sets varBuildWdkVer before build
```

Comment:
what is QB?

Reply:
CI Software - in fact the approach can be used locally as well.